dart/dynamics/CMakeLists.txt

```diff
@@ -8,6 +8,8 @@ dart_add_core_sources(${srcs} ${detail_srcs})
 
 # Generate header for this namespace
 dart_get_filename_components(header_names "dynamics headers" ${hdrs})
+# TODO: remove below line once the files are completely removed.
+list(REMOVE_ITEM header_names "MultiSphereShape.hpp")
```

Comment:
Yes, this excludes `MultiSphereShape.hpp` from the generated header `dart/dynamics/dynamics.hpp` so that it suppresses the warning of including the deprecated header.